tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -108,6 +108,14 @@ class TestExecutor(object):
     test_type = None
     convert_result = None
     supports_testdriver = False
+
+    # `supports_jsshell` determines whether a given test is enumerated in the
+    # various test-traversal commands exposed by the wptrunner CLI (see the
+    # `ignores` method). It is intended to be overridden by subclasses which
+    # are defined outside of WPT.
```

Comment:
That's not quite true in practice; [my code](https://searchfox.org/mozilla-central/rev/2466b82b729765fb0a3ab62f812c1a96a7362478/js/src/tests/jstests.py#353) doesn't use executors at all.